src/celeritas/random/CuHipRngEngine.hh

```diff
@@ -97,10 +100,13 @@ class GenerateCanonical<CuHipRngEngine, double>
  * Construct from state.
  */
 CELER_FUNCTION
-CuHipRngEngine::CuHipRngEngine(StateRef const& state, TrackSlotId const& id)
+CuHipRngEngine::CuHipRngEngine(ParamsRef const& params,
+                               StateRef const& state,
+                               TrackSlotId tid)
+    : params_(params)
```

Comment:
Is `params_` unused? You could just omit the local storage of it for clarity. The important thing of course is just that the interface is consistent across RNGs.